build/parseSpec.c

```diff
@@ -421,8 +422,8 @@ int readLine(rpmSpec spec, int strip)
 	    match = parseExpressionBoolean(s);
 	    if (match < 0) {
 		rpmlog(RPMLOG_ERR,
-			    _("%s:%d: bad %%if condition\n"),
-			    ofi->fileName, ofi->lineNum);
+			    _("%s:%d: bad %%if condition\n       expanded line:    %s"),
+			    ofi->fileName, ofi->lineNum, z);
```

Comment:
The common style is to just output the line after a colon and without ado about expansions, ie
```
_("%s:%d: bad %%if condition: %s")
```

Lets stick to that please. Couple of other nits: 
- why include the possible preceding whitespace in the error message? 
- the commit message talks about %elif message but support for that hasn't been added yet